src/templates/settings.html

```diff
@@ -24,7 +24,7 @@
   <div
     data-setting="export"
     class="tool-icon  icon-settings-export-white"
-    title="<span class='highlight'>EXPORT</span></br>Export as Image, as Spritesheet<br/>or as Animated GIF"
+    title="<span style='color:#7665a0'>EXPORT</span></br>Export as Image, as Spritesheet<br/>or as Animated GIF"
```

Comment:
this can't be done via stylesheet instead of inline?

Reply:
Good question! I was following the precedent of the 'Resize' icon here and some quick experimenting shows that doing this via stylesheet is non-trivial.

Reply:
That probably wasn't a great call on my part. Up to you what you think if going to be easier to merge in the future.

Reply:
I took a look again, and realized I had made a typo when trying to change to stylesheet. Incoming commit has this fix. Thanks for helping me figure it out!